sherpa/astro/ui/tests/test_astro_ui.py

```diff
@@ -29,7 +29,7 @@
 
 class test_ui(SherpaTestCase):
```

Comment:
It looks like `@requires_fits` should be moved to the `class test_ui` statement and removed from each method.